balancer/grpclb/grpclb_remote_balancer.go

```diff
@@ -103,6 +103,8 @@ func (lb *lbBalancer) processServerList(l *lbpb.ServerList) {
 // backendAddrs (whether any SubConn was newed/removed).
 // Caller must hold lb.mu.
 func (lb *lbBalancer) refreshSubConns(backendAddrs []resolver.Address, fromGRPCLBServer bool) {
+	lb.inFallback = !fromGRPCLBServer
```

Comment:
I think it is better to move the check for `lb.inFallback` inside `refreshSubConns`, so we don't need to check the condition every time when calling the `refreshSubConns`. This change is optional though.

Reply:
Some is `if fallback { refreshSubConns }`.
Some is `if !fallback { refreshSubConns }`.
How can those be moved inside?